### PR TITLE
Minor logging enhancement

### PR DESCRIPTION
**Pull request by granders**
@ewencp Addresses https://github.com/confluentinc/ducktape/issues/122

This cleans up the printout during test run a bit, and propagates test error to test_log.* and session_log.*

Also, removes the pesky "default" which kept appearing in jenkins console...

### CODE REVIEW

File: ducktape/tests/runner.py [hunk 1]
```diff
@@ -117,10 +117,10 @@ def run_all_tests(self):
                 result.stop_time = time.time()
                 self.results.append(result)
 
+                self.log(logging.INFO, "Summary: %s" % str(result.summary))
+                self.log(logging.INFO, "Data: %s" % str(result.data))
                 test_reporter = SingleResultFileReporter(result)
                 test_reporter.report()
-                test_reporter = SingleResultStdoutReporter(result)
```
Comment (ewencp) [inline ducktape/tests/runner.py]: At a minimum, there's an extraneous import now. But also, if we're not even creating this anymore, do we need to keep the class around at all?

**Comment by ewencp** [comment]
@granders LGTM. I'd also clean out the bit of code I pointed out if you can.

**Comment by ewencp** [comment]
Actually, I think I take that back. Having *some* delineation between tests was useful. Worst case, maybe just add a blank line after the `Data` line?

**Comment by granders** [comment]
@ewencp Added delimiter

**Comment by ewencp** [comment]
@granders much better, thanks.

**Pull request merged**